src/global.rs

```diff
@@ -1,5 +1,5 @@
-use std::rc::Rc;
-use std::cell::Cell;
+use alloc::rc::Rc;
+use core::cell::Cell;
```

Comment:
One thing we do in Cranelift is to remap `core` to `std` in the top-level lib.rs [here](https://github.com/CraneStation/cranelift/blob/master/lib/codegen/src/lib.rs#L110) so that most files can just use `std::`. This obviously isn't necessary, but since we have some crates that support no_std and some that don't, it's nice to the code more consistent between the two.

Reply:
This would have made rebasing a lot nicer, too :upside_down_face: